src/main/java/org/galatea/starter/service/ApiError.java

```diff
@@ -0,0 +1,21 @@
+package org.galatea.starter.service;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NonNull;
+import lombok.ToString;
+
+import org.springframework.http.HttpStatus;
+
+@Getter
+@ToString
+@AllArgsConstructor
+public class ApiError {
```

Comment:
This should be actually moved to the entrypoint package given that, like the RestExceptionHandler, it applies only to rest services.